hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTimelineArchiveLog.java

```diff
@@ -82,6 +90,7 @@
   private final int minInstantsToKeep;
   private final HoodieTable<T, I, K, O> table;
   private final HoodieTableMetaClient metaClient;
+  private final String mergeArchivePlanName = "mergeArchivePlan";
```

Comment:
Should this be a `public static final` variable?

Reply:
Changed. Just add `public static final String MERGE_ARCHIVE_PLAN_NAME = "mergeArchivePlan";` in `HoodieArchivedTimeline.java `.

Reply:
Sg